docs/src/external_mctp_vdm_cmds.md

```diff
@@ -0,0 +1,508 @@
+
+| Byte(s) | Name         | Type    | Description                                 |
+|---------|--------------|---------|---------------------------------------------|
+| 0:3     | length       | u32     | Length of the message in DWORDs             |
+| 4       | unlock_level | u8      | Debug unlock level (1-8)                    |
+| 5:7     | reserved     | u8[3]   | Reserved field                              |
+
+**Response Payload**:
+
+| Byte(s) | Name                    | Type      | Description                                         |
+|---------|-------------------------|-----------|-----------------------------------------------------|
+| 0:3     | completion_code         | u32       | Command completion status                           |
+| 4:7     | length                  | u32       | Length of the message in DWORDs                     |
+| 8:39    | unique_device_identifier| u8[32]    | Device identifier of the Caliptra device            |
+| 40:87   | challenge               | u8[48]    | Random number challenge                             |
+
+#### Authorize Debug Unlock Token
+
+Authorizes the debug unlock token.
+
+**Request Payload**:
+
+| Byte(s)   | Name                     | Type         | Description                                                                 |
+|-----------|--------------------------|--------------|-----------------------------------------------------------------------------|
+| 0:3       | length                   | u32          | Length of the message in DWORDs                                             |
+| 4:35      | unique_device_identifier | u8[32]       | Device identifier of the Caliptra device                                    |
+| 36        | unlock_level             | u8           | Debug unlock level (1-8)                                                    |
+| 37:39     | reserved                 | u8[3]        | Reserved field                                                              |
+| 40:87     | challenge                | u8[48]       | Random number challenge                                                     |
+| 88:183    | ecc_public_key           | u32[24]      | ECC public key in hardware format (little endian)                           |
+| 184:2635  | mldsa_public_key         | u32[648]     | MLDSA public key in hardware format (little endian)                         |
+| 2636:2731 | ecc_signature            | u32[24]      | ECC P-384 signature of the message hashed using SHA2-384 (R and S coordinates) |
+| 2732:6195 | mldsa_signature          | u32[1157]    | MLDSA signature of the message hashed using SHA2-512 (4627 bytes + 1 reserved byte) |
+
+**Response Payload**:
+
+| Byte(s) | Name            | Type | Description                |
+|---------|-----------------|------|----------------------------|
+| 0:3     | completion_code | u32  | Command completion status  |
+
+## Unified Handling of Overlapping MCTP VDM External Commands and MCI Mailbox Commands
```

Comment:
This looks great. I think we could move this part out to a separate specification, and have VDM and MCI mailbox specs as sub-headers of this.